handlers/PivotalHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { subDays } from 'date-fns';
+
+export const STORY_STATES = ['unscheduled', 'unstarted', 'started', 'finished', 'delivered', 'rejected', 'accepted'];
+const PIVOTAL_API_URL = 'https://www.pivotaltracker.com/services/v5';
+
+class PivotalHandler {
+  // Gets all projects for the provided user apiKey.
+  static async fetchProjects({ apiKey }) {
+    const response = await fetch('https://www.pivotaltracker.com/services/v5/projects', {
+      headers: {
+        'X-TrackerToken': apiKey,
+      },
+    });
+
+    return await response.json();
+  }
+
+  // Gets all project stories for the provided user apiKey.
+  static async fetchProjectStories({ apiKey, projectId }) {
+    const stories = await Promise.all(
+      STORY_STATES.map(async state => {
+        let fetchString = `stories?limit=500&with_state=${state}&fields=name,estimate,owners,labels,blockers,reviews,story_type,description`;
+        if (state === 'Accepted') {
+          const oneWeekAgo = subDays(new Date(), 7);
+          fetchString = `${fetchString}&accepted_after=${oneWeekAgo.getTime()}`;
+        }
+
+        const request = await fetch(`https://www.pivotaltracker.com/services/v5/projects/${projectId}/${fetchString}`, {
+          headers: {
+            'X-TrackerToken': apiKey,
+          },
+        });
+        return { [state]: await request.json() };
```

Comment:
Hey! Nice PR!! 
I hadn't seen it and also tackled the same issue of using `Promise.all`. 
I think in your case you are using an `await` on line 28 that shouldn't be there if you want the requests to be done in parallel. 
You are free to steal the code I put here https://github.com/opticpower/tracker/pull/24 to make the requests parallel

Reply:
Hey @mikealche, I cant remove that `await` there because then the `json` parse will not be allowed. 

<img width="1122" alt="Screen Shot 2020-10-13 at 1 19 52 PM" src="https://user-images.githubusercontent.com/42581689/95894784-95b64b80-0d57-11eb-8400-c5ea651dffd3.png">


Maybe is better with `axios` because you don't have to do another asynchronous task to get the `json` from the response.